capture/src/capture.rs

```diff
@@ -38,6 +38,7 @@ use crate::{
         compression
     )
 )]
+#[debug_handler]
```

Comment:
shows more informative compiler errors. Allowed me to pinpoint that the issue was `InsecureClientIp` needed an upgrade

Reply:
> This macro has no effect when compiled with the release profile.

Nice